commcare_connect/opportunity/views.py

```diff
@@ -2256,7 +2256,7 @@ def opportunity_delivery_stats(request, org_slug, opp_id):
             "name": "Services Delivered",
             "status": "Total",
             "value": header_with_tooltip(stats.total_deliveries, "Total delivered so far excluding duplicates"),
-            "url": delivery_url,
+            "url": delivery_url + "?sort=-last_active",
```

Comment:
same comment I always make in these cases but can we use a url generation library to pass these params in rather than using string concatenation. I know it recently caused a bug when we changed a base url, and that would make it much harder to break (and the code feel a bit cleaner). This applies to all the urls where we add params by string concatenation, including the one in the template